api/src/utils/file-utils.ts

```diff
@@ -1,7 +1,11 @@
 import AWS from 'aws-sdk';
 import { DeleteObjectOutput, GetObjectOutput, ManagedUpload, Metadata } from 'aws-sdk/clients/s3';
 import { S3_ROLE } from '../constants/roles';
+import clamd from 'clamdjs';
 
+const scanner = process.env.SHOULD_CLAMAV_SCAN
+  ? clamd.createScanner(process.env.CLAMAV_HOST || 'clamav', Number(process.env.CLAMAV_PORT) || 3310)
```

Comment:
Do we need the hardcoded defaults?

If someone has `SHOULD_CLAMAV_SCAN` enabled, Id think its safe to assume they will also have the HOST/PORT variables set.

Reply:
we want it hard coded default so that it just always works in Openshift even without the env variables cause the values will never change regardless of environment